src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
```

Comment:
The function name 'makeCalculator' is appropriate as it clearly describes the action of the function, which is to create a 'calculator' object.